client/Dockerfile

```diff
@@ -1,13 +1,17 @@
 # pull official node base image
 FROM node:16.2.0-alpine3.12 as build
+
 WORKDIR /app
 ENV PATH /app/node_modules/.bin:$PATH
 COPY package.json ./
 COPY package-lock.json ./
-RUN npm ci --silent
-RUN npm install react-scripts@3.4.1 -g --silent
+
+# TODO Either copy file individually or use this line + .gitignore
```

Comment:
As Thomas and I just discussed, this should probably follow the example of one of the example annotators, e.g. https://github.com/nlpsandbox/person-name-annotator-example/blob/8423e95c924d1232f68560127815dacfd3a4ccd0/server/Dockerfile#L19

("`.gitignore`" should be "`.dockerignore`")